Error in set-up of application BADBINARY - /non_existent_file.py has not been built.
Using Application ENV, checkout /users/geoff/texttesttmp/texttestgeoff16Jun18:14:34/TestSelf/EnvironmentFile/RemoveVarOnException/TargetApp/./
Running ENV test-suite TargetApp
  Running ENV test-case Test2
Comparing differences for ENV test-suite TargetApp
  Comparing differences for ENV test-case Test2 - SUCCESS! (on output.env)
